File: Frontend/debtmanage/src/app/debts/debt.ts
export class Debt{
    private _id: string;
    private _idUser: number;
    private _motivo: string;
    private _valor: number;
    private _data: number;

    public get idUser(): number {
        return this._idUser;
    }
    public set idUser(value: number) {
        this._idUser = value;
    }
    
    public get motivo(): string {
        return this._motivo;
    }
    public set motivo(value: string) {
        this._motivo = value;
    }
    
    public get valor(): number {
        return this._valor;
    }
    public set valor(value: number) {
        this._valor = value;
    }
    
    public get data(): number {
        return this._data;
    }
    public set data(value: number) {
        this._data = value;
    }

    public get id(): string {
        return this._id;
    }
    public set id(value: string) {
        this._id = value;
    }

}

File: Backend/api/src/main/java/com/teste/neliogodoi/fullstack/model/Debt.java
package com.teste.neliogodoi.fullstack.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import org.springframework.data.mongodb.core.mapping.Field;


@Document(collection = "divida")
public class Debt {
	
	@Id
	private String id;
	
	@Field("id_user")
	private long idUser;
	
	@Field("motivo")
	private String motivo;
	
	@Field("valor")
	private Double valor;
	
	@Field("data")
	private long data;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public long getIdUser() {
		return idUser;
	}

	public void setIdUser(long idUser) {
		this.idUser = idUser;
	}

	public String getMotivo() {
		return motivo;
	}

	public void setMotivo(String motivo) {
		this.motivo = motivo;
	}

	public Double getValor() {
		return valor;
	}

	public void setValor(Double valor) {
		this.valor = valor;
	}

	public long getData() {
		return data;
	}

	public void setData(long data) {
		this.data = data;
	}
		
}
